python/examples/data_import/csv/simple/main.py

```diff
@@ -0,0 +1,32 @@
+import os
+
+from dotenv import load_dotenv
+from sift_py.data_import.csv import CsvUploadService
+from sift_py.rest import SiftRestConfig
+
+if __name__ == "__main__":
+    """
+    Example of uploading a CSV file into Sift.
+    """
+
+    load_dotenv()
+
+    sift_uri = os.getenv("SIFT_API_URI")
+    assert sift_uri, "expected 'SIFT_API_URI' environment variable to be set"
+
+    apikey = os.getenv("SIFT_API_KEY")
+    assert apikey, "expected 'SIFT_API_KEY' environment variable to be set"
+
+    asset_name = os.getenv("ASSET_NAME")
+    assert asset_name, "expected 'ASSET_NAME' environment variable to be set"
+
+    rest_config: SiftRestConfig = {
+        # Be sure to exclude the "https://" or "http://" scheme out of the uri
```

Comment:
we can remove this if we implement my other comment about `_clean_uri` from the `lib/sift_py/grpc/transport.py` module.